rules.py

```diff
@@ -826,7 +831,53 @@ def check(self):
         if error_log:
             raise base.ElectionTreeError(
                 "The Election File contains unreferenced Candidates", error_log)
-                
+
+
+class CheckIdentifiers(base.TreeRule):
+    """Check that the NIST objects in the feed has an '<ExternalIdentifier>' block.
+        Add error message if the block is missing."""
+
+    def check(self):
+        identifier_values = {}
+        error_log = []
+        nist_objects = ('Candidate', 'Contest', 'Party')
```

Comment:
Nit: Use " instead of ' as quote marks here since that's how the rest of the file does it.